applications/RANSApplication/custom_conditions/data_containers/k_epsilon/epsilon_k_based_wall_condition_data.h

```diff
@@ -53,24 +53,27 @@ class EpsilonKBasedWallConditionData : public ScalarWallFluxConditionData
     }
 
     EpsilonKBasedWallConditionData(
-        const GeomtryType& rGeometry)
-    : BaseType(rGeometry)
+        const GeometryType& rGeometry,
+        const Properties& rProperties,
+        const ProcessInfo& rProcessInfo)
+    : BaseType(rGeometry, rProperties, rProcessInfo)
     {
     }
 
     void CalculateConstants(
-        const ProcessInfo& rCurrentProcessInfo) override;
+        const ProcessInfo& rCurrentProcessInfo);
 
-    bool IsWallFluxComputable() const override;
+    bool IsWallFluxComputable() const;
 
     double CalculateWallFlux(
-        const Vector& rShapeFunctions) const override;
```

Comment:
I guess you removed the `override`s to avoid passing through the virtual table. However, I'm not completely sure if this would work. If the base class method is `virtual` I don't think it won't automatically devirtualize the method. AFAIK `override` was introduced in c++11 as a safe-ward to avoid accidental method overload.

Reply:
I dont have these methods now in the base class, so c++ compiler cannot create these methods virtual in the first place ryt?, so devirtualization is not required since virtual methods were not present in first place. I wanted to put these proper method selection burden on compiler time rather than run time even if it is a small gain :). Do you think still it does not make sense?

Reply:
Sorry. I thought these were present in the base class. Forget about my comment. If there is no virtual method the discussion makes no sense.